### ISSUE TITLE
support customization of emoji appearance [use case: add hover text with keyword when shown in browser]

### ISSUE THREAD

**Issue by pietroppeter** [pietroppeter/nimoji#4]
(optional) provide emoji keyword as alt text (as inline html)

**Comment by pietroppeter** [comment]
trying to find out what would be the appropriate html element I realized alt text is not it (supported on very little tags and not standard compliant, see https://en.wikipedia.org/wiki/Alt_attribute). Instead it is `title` attribute that should be used, and I guess that tag should be a simple `span`.

**Comment by pietroppeter** [comment]
as far as nimoji api is concerned, since this is a feature that is probably used consistently in a codebase, it could be implemented as a compile time symbol. Name of compile time symbol? `nimojiHtml`? `nimojiTooltip`? `nimojiAddTitleAttribute`?

default of course is not having it.

**Comment by pietroppeter** [comment]
my confusion was probably tied to this post: https://www.reddit.com/r/firefox/comments/5wxgwl/alttext_for_emoji/ which produces title effect with markdown link and alt text. This is another option (also interesting if you want to add a link. I am actually thinking that probably the best way is to have an alternative emojize proc that also returns positions of emojis in text. This could be useful for many things including adding any html inline or markdown element that ones want.
